### ISSUE TITLE
Android log folder needs changed. (Softlocked)

### ISSUE THREAD

**Issue by Mou-Ikkai** [ppy/osu#21076]
### Type

Game behaviour

### Bug description

Android 12 and 13 seem to completely prohibit access to the Android/Data folder, even with Show Hidden Files enabled. As such, we can't make proper bug reports. No, other file explorers can't access the folder, as the Files app stupidly says "To protect your privacy, choose another folder."

### Screenshots or videos

![Screenshot_20221102-124939](https://user-images.githubusercontent.com/45863583/199552117-59801e2c-0ae0-44e0-98b8-4883a31d76c7.png)
![Screenshot_20221102-124945](https://user-images.githubusercontent.com/45863583/199552164-52719dc3-6128-4d67-a43b-e387191713b3.png)

### Version

2022.1101.0-lazer

### Logs

Softlocked out of log folder.

**Comment by Mou-Ikkai** [comment]
Few suggestions: Android/Media or Android/Obb

**Comment by Mou-Ikkai** [comment]
> You can use another file manager. You cannot modify the data but you can copy the files.

No... You cannot. As I showed in the 2nd image, your file explorer has to ask for access, but android declines and tells you to "choose another folder".

**Comment by bdach** [comment]
This actually changed in [android 11](https://developer.android.com/about/versions/11/privacy/storage).

The files are still accessible if you, for instance, plug the phone into a computer and access the log files through it. Doing *something* about this may not be a bad idea, but I'm not sure what can be done without breaking google's explicit wish to scope down app storages.
